sdk/communication/Azure.Communication.Administration/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
 ### Added
```

Comment:
We had breaking changes and added LRO support right? Let's mention this here

Reply:
@DominikMe  do you know who mostly worked on these changes? I'll reach out for more details.

Reply:
Nvm.. checking history for PRs thanks!